lib/ProductOpener/Ingredients.pm

```diff
@@ -1221,6 +1223,27 @@ sub parse_origins_from_text ($product_ref, $text) {
 	return $text;
 }
 
+=head2 get_percent_or_quantity_and_normalized_quantity($percent_or_quantity_value, $percent_or_quantity_unit)
+
+Used to assign percent or quantity for strings parsed with $percent_or_quantity_regexp.
+
```

Comment:
can you add a small example ?

Reply:
sure, I added more doc and an example, also I found that I wasn't using this function everywhere I wanted to, so that's fixed.